Dockerfile

```diff
@@ -1,149 +1,172 @@
-# To build with OpenMC
+# To build with OpenMC and by default this Dockerfile builds the master branch of OpenMC.
 # docker build -t openmc .
 
+# To build with OpenMC develop branch
+# docker build -t openmc_develop --build-arg openmc_branch=develop .
+
 # To build with OpenMC and DAGMC enabled
-# docker build -t openmc_dagmc --build-arg include_dagmc=true .
+# docker build -t openmc_dagmc --build-arg build_dagmc=on --build-arg compile_cores=4 .
+
+# To build with OpenMC and Libmesh enabled
+# docker build -t openmc_libmesh --build-arg build_libmesh=on --build-arg compile_cores=4 .
+
+# To build with both DAGMC and Libmesh enabled
+# docker build -t openmc_dagmc_libmesh --build-arg build_dagmc=on --build-arg build_libmesh=on --build-arg compile_cores=4 .
 
-# To make use of multiple cores during the compile stages of the docker build
-# docker build -t openmc_dagmc --build-arg compile_cores=8 .
+# sudo docker run image_name:tag_name or ID with no tag sudo docker run ID number
 
 FROM ubuntu:latest
```

Comment:
```suggestion
FROM ubuntu:20.04
```

Reply:
Perhaps we should pin the ubuntu version which might help avoid issues like #1897

Reply:
It looks like the `latest` tag points to the latest LTS release, i.e. 20.04. I'm not seeing how this would help issue #1897.